analyzer/restapi-plugin/src/main/java/eu/fasten/analyzer/restapiplugin/mvn/LazyIngestionProvider.java

```diff
@@ -60,7 +67,7 @@ public static void ingestArtifactIfNecessary(String packageName, String version,
                 jsonRecord.put("date", date);
             }
             Timestamp curTime = new Timestamp(System.currentTimeMillis());
-			KnowledgeBaseConnector.kbDao.insertIngestedArtifact(packageName, version, curTime);
+            KnowledgeBaseConnector.kbDao.insertIngestedArtifact(toKey(packageName, version), "0.0.1-SNAPSHOT");
```

Comment:
Why do we have `0.0.1-SNAPSHOT`?

Reply:
We do not have a proper versioning strategy and `0.0.1-SNAPSHOT` is used as the version in all poms of the old server, so I felt it is appropriate to use it here :) The main user of said methods is the new `PomAnalyzer`, which will provide proper versions.

The concrete value is also not that important and might only become interesting in the future. The intention was to give us the means to void ingestions that have been done with a particular version, should we detect a bug in said version. What we will be probably be doing is to just wipe the table every now and then, so the version might be irrelevant.